services/tenant-ui/frontend/src/matomoSetup.ts

```diff
@@ -0,0 +1,18 @@
+//@ts-ignore
+var _paq = (window._paq = window._paq || []);
```

Comment:
Not pretty but can handle the matomo added `_paq` with
`const _paq = ((window as any)._paq = (window as any)._paq || []);`

Does that break anything?

Reply:
That works great, thanks Lucas